### PR TITLE
ContentRoot updates for .NET 7 /7

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/host-and-deploy/windows-service?view=aspnetcore-7.0&branch=pr-en-us-26913&tabs=visual-studio#app-configuration)

Per [this comment](https://github.com/dotnet/AspNetCore.Docs/pull/26844#pullrequestreview-1092443968)
> This looks good to me. In .NET 7, you will no longer have to manually change `WebApplicationOptions.ContentRootPath` in windows services because of [dotnet/runtime#68580](https://github.com/dotnet/runtime/pull/68580).
> 
> That PR also adds `builder.Services.AddWindowsService()` which does the same thing as `builder.Host.UseWindowsService()` without using the `builder.Host`, `IHostBuilder` compatibility layer but with less indirection. `AddWindowsService()` also works with the `HostApplicationBuilder` we added in .NET 7.

## Public review  of the doc build:
- download the [HTML files](https://www.dropbox.com/sh/yy994551cje0kap/AAB0tRnm7gze2kYrSYPSFHIda?dl=0)
- unzip
- view HTML file in a browser.
- For links outside the doc, remove the `review.` from URL.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@martincostello please review changes and public [HTML file](https://www.dropbox.com/sh/yy994551cje0kap/AAB0tRnm7gze2kYrSYPSFHIda?dl=0)

File: aspnetcore/host-and-deploy/windows-service/samples/7.x/WebAppServiceSample/Services/ServiceA.cs [hunk 1]
```diff
@@ -1,34 +1,27 @@
-using System;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
+namespace SampleApp.Services;
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/windows-service/samples/7.x/WebAppServiceSample/Services/ServiceA.cs:1]: I'm not sure why the diff file is so awful, I just used the new namespace declaration and removed implicit using's.

**Review by martincostello — APPROVED** [review]
Seems OK to me.

File: aspnetcore/host-and-deploy/windows-service/samples/7.x/WebAppServiceSample/Services/ServiceA.cs (hunk 1 above)
Comment (martincostello) [reply aspnetcore/host-and-deploy/windows-service/samples/7.x/WebAppServiceSample/Services/ServiceA.cs:1]: Yeah, git always does this going to file-scoped namespaces. Turning off whitespace in the diff sorts it out mostly.

**Pull request merged**